package.json

```diff
@@ -35,6 +35,7 @@
         "@types/jest": "^26.0.24",
         "@types/jsdom": "12.2.2",
         "@types/lodash": "4.14.121",
+        "@types/node": "^22.8.4",
```

Comment:
Some changes were running fine locally and hitting type errors when our github actions tried to run type checks, this fixed it.

Reply:
Good question.